app/views/layouts/application.html.erb

```diff
@@ -10,10 +10,20 @@
 
   <body>
     <nav class = "topnav">
+
       <%= link_to "All Merchants", "/merchants"%>
       <%= link_to "All Items", "/items"%>
-      <%= link_to "Register", "/register"%>
+      <%= link_to "Profile", "/profile"%>
+      <% if !current_user %>
```

Comment:
Loving this switch statement to ensure our logged in users only see the links that make sense